magento/magento2
# Allow filter directive processors to be used with the Widget template filter

### Description (*)
This allows the use of filter directives with the `Magento\Widget\Model\Template\Filter` filter (which is used by CMS content). This makes extending the filter directives actually useful.

By applying the changes in this Pull Request, the following DI configuration also applies to this filter as well:
```xml
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
    <type name="Magento\Framework\Filter\Template">
        <arguments>
            <argument name="directiveProcessors" xsi:type="array">
                <item name="my-custom-directive" sortOrder="1000" xsi:type="object"></item>
            </argument>
        </arguments>
    </type>
</config>
```

### Manual testing scenarios (*)
1. Create a custom filter directive via DI
2. See this filter directive now being applied for CMS pages and CMS blocks

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29764: Allow filter directive processors to be used with the Widget template filter
